be/src/exec/pipeline/sink/file_sink_operator.cpp

```diff
@@ -163,6 +167,7 @@ Status FileSinkOperator::prepare(RuntimeState* state) {
 }
 
 void FileSinkOperator::close(RuntimeState* state) {
+    _file_sink_buffer->close(state);
```

Comment:
FileSinkOperator::close will be invoked in pipeline execution thread, FileSinkIOBuffer::close may contain sync io reqeust in writer->close, we should avoid this.